util-node/getBannersData.js

```diff
@@ -0,0 +1,17 @@
+const fetch = require('node-fetch');
+
+async function getBannersData() {
+  try {
+    const siteResponse = await fetch(
+      'https://raw.githubusercontent.com/nodejs/nodejs.org/master/locale/en/site.json'
```

Comment:
Couldnt get the mocks to work for this call and the calls in `getNodeReleasesData`. Idk what i could be doing wrong though. I ended up deleting the tests. Maybe they can be a future PR